### ISSUE TITLE
var foo = require('bar'), fs = require('fs'); doesn't parse

### ISSUE THREAD

**Issue by gaye** [substack/brfs#28]
```
gareth@samson:~/Documents/davinci$ make
./node_modules/.bin/browserify -t brfs ./lib/index.js > ./davinci.js
SyntaxError: Line 2: Unexpected string while parsing /home/gareth/Documents/davinci/lib/template/index.js
    at Stream.end (/home/gareth/Documents/davinci/node_modules/browserify/node_modules/insert-module-globals/index.js:71:21)
    at _end (/home/gareth/Documents/davinci/node_modules/browserify/node_modules/insert-module-globals/node_modules/through/index.js:65:9)
    at Stream.stream.end (/home/gareth/Documents/davinci/node_modules/browserify/node_modules/insert-module-globals/node_modules/through/index.js:74:5)
    at DuplexWrapper.onend (/home/gareth/Documents/davinci/node_modules/brfs/node_modules/static-module/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_readable.js:530:10)
    at DuplexWrapper.g (events.js:199:16)
    at DuplexWrapper.EventEmitter.emit (events.js:129:20)
    at /home/gareth/Documents/davinci/node_modules/brfs/node_modules/static-module/node_modules/duplexer2/node_modules/readable-stream/lib/_stream_readable.js:927:16
    at process._tickCallback (node.js:343:11)
make: *** [davinci.js] Error 1
```
Fixed by moving the call to `require('fs')` onto its own line.

**Comment by mabbage** [comment]
+1 Am experiencing this too

**Comment by mikemaccana** [comment]
Can confirm this has bit me too, have had to comment the behaviour to stop my colleagues breaking stuff:

    // Note this can't be merged into another var statement, due to the way brfs works.
    var fs = require("fs");

**Comment by sterpe** [comment]
Ah okay, I just discovered this too.  +1

**Comment by mattdesl** [comment]
This seems subtly different from #25 (which I can't reproduce). 

Oddly enough; if the transform returns a string instead of a stream, it works as expected. Example transform which works as expected:

```js
function readFileSync(file) {
  return '"' + fs.readFileSync(file, 'utf8') + '"'
}
```

Switching it to a stream will introduce the error again:

```js
function readFileSync(file) {
  return fs.createReadStream(file, 'utf8')
    .pipe(quote())
}
```

Testing with static-module@1.1.2, brfs@1.4.0, browserify@9.0.3 and the following code:

```js
var fs = require('fs'),
  test = require('./test')

var data = fs.readFileSync('./data.txt', 'utf8')
console.log(data)
```

**Comment by inca** [comment]
:+1: Good catch, spent almost 15 mins figuring this one out :)

**Comment by lvarayut** [comment]
I'm also having the same issue using the latest version. Just spend an hour to figure this out.

**Comment by sprat** [comment]
Same for me

**Comment by mattdesl** [comment]
This should be fixed now in `static-module@1.3.0`, so it should work next time you install `brfs`. :tada:

Many thanks to @eush77 for the PR!

**Comment by mikemaccana** [comment]
Woo @mattdesl and thanks @eush77!

For everyone else: just reinstalling brfs seems to grab the updated dependency:

    $ npm ls static-module
    certsimple@1.0.0 /Users/mike/Documents/certsimple
    └─┬ brfs@1.4.2
      └── static-module@1.1.3

    $ npm install --save brfs
    certsimple@1.0.0 /Users/mike/Documents/certsimple
    └─┬ brfs@1.4.2
      └── static-module@1.3.0
